doc/conf.py

```diff
@@ -64,7 +66,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'Luma.OLED: Display drivers for SSD1306 / SSD1322 / SSD1325 / SSD1331 / SSD1351 / SH1106'
+project = 'Luma.OLED: Display drivers for {0}'.format(
+    '/ '.join(supported_devices).upper())
```

Comment:
Maybe just use a comma like in the other PR

Reply:
ok cool

Reply:
I also hope it solves the missing device on https://luma-oled.readthedocs.io/en/latest/api-documentation.html#module-luma.oled although I'm not sure where that line's coming from..

```
OLED display driver for SSD1306, SSD1322, SSD1325, SSD1331 and SH1106 devices.
```